Configuration/ProcessModifiers/python/kf_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is for injecting KF-based iterations in TICL.
+
+kf =  cms.Modifier()
```

Comment:
I suggest changing the name of the Modifier to make it more explicit, something along the lines of: 
```suggestion
hgcal_ticl_kf =  cms.Modifier()
```

Reply:
Remember when you open the PR to include a line with a recipe to test the Modifier for the reviewers.

Reply:
Ok sounds good. But separate from the code right? So it's gonna be in the comment section along the lines of "Test modifier on xxxxx.x" right?

Reply:
exactly